### ISSUE TITLE
Cannot import latest release (10.0.0) 

### ISSUE THREAD

**Issue by KirkHadley** [splitio/python-client#539]
I get the following error when attempting `from splitio import get_factory` using the latest version (10.0.0): 
```
/usr/local/lib/python3.11/site-packages/splitio/__init__.py:1: in <module>
    from splitio.client.factory import get_factory, get_factory_async
/usr/local/lib/python3.11/site-packages/splitio/client/factory.py:57: in <module>
    from splitio.sync.manager import Manager, RedisManager, ManagerAsync, RedisManagerAsync
/usr/local/lib/python3.11/site-packages/splitio/sync/manager.py:8: in <module>
    from splitio.push.manager import PushManager, PushManagerAsync, Status
/usr/local/lib/python3.11/site-packages/splitio/push/manager.py:5: in <module>
    from splitio.optional.loaders import asyncio, anext
E   ImportError: cannot import name 'anext' from 'splitio.optional.loaders' (/usr/local/lib/python3.11/site-packages/splitio/optional/loaders.py)
```

The error is fixed by pinning to 9.7.0.

**Comment by KirkHadley** [comment]
FYI- bumping the minor version in this [line](https://github.com/splitio/python-client/blob/master/splitio/optional/loaders.py#L20) to 12 (I'm on python 3.11) also appears to fix this

**Comment by chillaq** [comment]
Thanks @KirkHadley for the info, will check it shortly

**Comment by chillaq** [comment]
Hi @KirkHadley, we released version 10.0.1 for the fix.

Thanks
Bilal
